modules/lambda-registrator/README.md

```diff
@@ -0,0 +1,7 @@
+# Lambda Registrator
+
+This module creates a service that automatically registers and de-registers AWS Lambda functions
```

Comment:
In terms of wording, this module actually creates a Lambda function that does this. I think "service" is a bit ambiguous in this context.

Reply:
I agree. I think something like this gives a complete story without getting too long: This module deploys a Lambda function that automatically registers and de-registers AWS Lambda functions into Consul as services based on the Lambda function's tags."